plugins/obs-qsv11/CMakeLists.txt

```diff
@@ -58,5 +58,5 @@ elseif(OS_LINUX)
 
   target_sources(obs-qsv11 PRIVATE common_utils_linux.cpp)
 
-  target_link_libraries(obs-qsv11 PRIVATE Libva::va Libva::drm)
+  target_link_libraries(obs-qsv11 PRIVATE Libva::va Libva::drm Libva::wayland)
```

Comment:
Libva has it's own X11 library like DRM and Wayland.
To avoid linkage issue it should be added too.

Reply:
It doesnt provide any actual linkages, vaGetDisplay is X11 by definition. I can remove the X11 header if we want to be clear about it.

Reply:
Sorry poked the libraries a bit and yea we should link it, ill add it in a bit.